novatel_gps_driver/src/nodelets/novatel_gps_nodelet.cpp

```diff
@@ -256,10 +264,15 @@ namespace novatel_gps_driver
       reset_service_ = priv.advertiseService("freset", &NovatelGpsNodelet::resetService, this);
 
       sync_sub_ = swri::Subscriber(node, "gps_sync", 100, &NovatelGpsNodelet::SyncCallback, this);
-
+      //System alert subcriber object decleration
+      alert_sub = swri::Subscriber(node,"system_alert",100,&NovatelGpsNodelet::alertCallback,this);
```

Comment:
The queue size for system_alert can be 10 instead of 100 as their aren't many system alert messages that get sent at any one time.